src/platform/utilities/api/index.js

```diff
@@ -1,8 +1,68 @@
 import * as Sentry from '@sentry/browser';
 
 import environment from '../environment';
+import ENVIRONMENTS from '../../../site/constants/environments';
 import localStorage from '../storage/localStorage';
 
+let sessionTimeoutMinutesLengthSSO = 0;
+const SSOeEndpoint = () => {
+  let environmentPrefix;
+  switch (ENVIRONMENTS.BUILDTYPE) {
+    case ENVIRONMENTS.VAGOVSTAGING:
+      environmentPrefix = 'sqa.';
+      break;
+    case ENVIRONMENTS.VAGOVDEV:
+      environmentPrefix = 'int.';
+      break;
+    default:
+      environmentPrefix = '';
+  }
+
+  return `https://${environmentPrefix}eauth.va.gov/keepalive`;
+};
```

Comment:
I think a switch is a bit overkill here.  Something like this seems more appropriate:

```js
const SSOeEndpoint = () => {
  const environmentPrefixes = {
    [ENVIRONMENTS.VAGOVSTAGING]: 'sqa',
    [ENVIRONMENTS.VAGOVDEV]: 'int',
  }
  const envPrefix = environmentPrefixes[ENVIRONMENTS.BUILDTYPE] || ""

  return `https://${envPrefix}eauth.va.gov/keepalive`;
};
```

Reply:
good call, mapping those values out seems cleaner to me, as well